appveyor.yml

```diff
@@ -1,20 +1,39 @@
+# Notes:
+#   - Minimal appveyor.yml file is an empty file. All sections are optional.
+#   - Indent each level of configuration with 2 spaces. Do not use tabs!
+#   - All section names are case-sensitive.
+#   - Section names should be unique on each level.
+
 version: "{build}"
+
 os: Windows Server 2012 R2
+
+clone_folder: c:\go\src\github.com\ipfs\go-ipfs
+
 environment:
   GOPATH: c:\gopath
   TEST_NO_FUSE: 1
   TEST_VERBOSE: 1
   TEST_SUITE: test_sharness_expensive
+  GOFLAGS: -tags nofuse 
+  matrix:
+  - GOARCH: amd64
+    GOVERSION: 1.5.1
+    GOROOT: c:\go
+    DOWNLOADPLATFORM: "x64"
+  
 install:
-  - set PATH=%GOPATH%\bin;c:\go\bin;%PATH%
-  - rmdir c:\go /s /q
-  - appveyor DownloadFile https://storage.googleapis.com/golang/go1.5.1.windows-amd64.msi
-  - msiexec /i go1.5.1.windows-amd64.msi /q
+  # Enable make
+  - SET PATH=c:\MinGW\bin;%PATH%
+  - copy c:\MinGW\bin\mingw32-make.exe c:\MinGW\bin\make.exe
   - go version
   - go env
+  
 build_script:
-  - make install
+  - make nofuse
+  
 test_script:
-  - make $TEST_SUITE
+  - make test_sharness_expensive
```

Comment:
Actually, since go builds just fine, `go test -tags nofuse ./...` can be added before running the sharness test. However, expect to see lots of goroutine errs.